src/Datadog.Trace/Tracer.cs

```diff
@@ -556,8 +562,11 @@ internal async Task WriteDiagnosticLog()
                     writer.WritePropertyName("service");
                     writer.WriteValue(DefaultServiceName);
 
-                    writer.WritePropertyName("agent_url");
-                    writer.WriteValue(Settings.AgentUri);
+                    if (!AzureAppServices.Metadata.IsRelevant)
+                    {
+                        writer.WritePropertyName("agent_url");
+                        writer.WriteValue(Settings.AgentUri);
```

Comment:
Is there a reliable way to tell if the value was set to a custom value? If the Azure App Service is configured to send to an Agent on another machine via an IP address, it would be useful to print out.

Reply:
I think that setup is actually discouraged now, but I will follow up on this.

Reply:
Actually, it's not harmful to log this, it just won't have a value.
After named pipes is merged I'll check for the transport and instead log whatever is dictated by that.
The important part here is not failing the initial request and logging API key.